electron/main.ts

```diff
@@ -487,6 +489,7 @@ ipcMain.handle(
     const newProtonWinePath = wine.replace("proton'", "files/bin/wine64'")
     const oldProtonWinePath = wine.replace("proton'", "dist/bin/wine64'")
     const isProton = wine.includes('proton')
```

Comment:
Off-topic todo: Update this to use `WineInstallation` and with that the `type` attribute